CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.1.1+1
```

Comment:
```suggestion
## 4.1.2
```

Even without code changes we bump the patch version for doc or pubspec changes.

https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance#bumping-versions

Reply:
4.1.2 has already shipped, and in fact does contain pubspec changes, which @jonahwilliams and flutter_tools do not want.

Reply:
Ah I see.

In that case `4.1.1+1` is OK.